<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>757</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Ventana Principal</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tb_menu">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>301</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="t_abrir">
     <attribute name="title">
      <string>Imagen</string>
     </attribute>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>251</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="bt_abrir">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>215</y>
         <width>91</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Obtener</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb_archivo">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>30</y>
         <width>92</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Archivo</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb_dispositivo">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>50</y>
         <width>92</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Dispositivo</string>
       </property>
      </widget>
      <widget class="QLabel" name="lb_origen">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Origen:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lb_resolucion">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>103</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Resoluci√≥n:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cb_resoluciones">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>91</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cb_dispositivos">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>70</y>
         <width>61</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tb_edicion">
     <attribute name="title">
      <string>Edicion</string>
     </attribute>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>251</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="bt_transforma">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>91</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Transforma</string>
       </property>
      </widget>
      <widget class="QPushButton" name="bt_giroHor">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>91</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Giro Horizontal</string>
       </property>
      </widget>
      <widget class="QPushButton" name="bt_procesa">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>91</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Procesado</string>
       </property>
      </widget>
      <widget class="QPushButton" name="bt_exportar">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>215</y>
         <width>91</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Exportar</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="fr_imagen">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>921</width>
      <height>691</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lb_imagen">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>7</y>
       <width>900</width>
       <height>675</height>
      </rect>
     </property>
     <property name="text">
      <string>.</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
